### DISCUSSION TITLE
Document about scopes

### DISCUSSION THREAD

**Discussion by xdxxxdx** [eclipse-edc/.github#100] [Q&A] [answered]
Hello EDC team,

I see there is a new sample implemented to explain how policies are configured in the scope of edc. Thanks so much for it, which is very useful.
I am wondering if there is documentation somewhere to have a full list of scopes.
I can see there are: 

NEGOTIATION_SCOPE
CATALOGING_REQUEST_SCOPE
RESOLUTION_SCOP
BEARER_ACCESS_SCOPE
ALL_SCOPE
CATALOGING_SCOPE

Thanks

**Comment by mspiekermann** [comment, marked as answer]
There is the documentation of the [Poliy Engine](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/policy-engine.md), which (should) give an overview about the scopes. Creating the new sample(s) is part of our goal to update and improve the documentation for policy handling. If you are investigating further and find additions to the current documentation, please feel free to file a PR.

**Reply by xdxxxdx** [reply]
Thanks so much for your reply @mspiekermann

**Comment by xdxxxdx** [comment]
@Yalz FYI
